lib/fabric/app/models/fabric/charge.rb

```diff
@@ -105,11 +105,15 @@ def sync_external(charge)
         fbt.sync_with(sbt).save
       end
 
-      refunds = charge.refunds.data
-      if refunds.size.positive? && self.refunds.size != refunds.size
-        refunds.each do |sr|
-          fr = Fabric::Refund.find_or_initialize_by(stripe_id: sr.id)
-          fr.sync_with(sr).save
+      # refunds are no longer included by default. don't want this to break
+      # if we get here from somewhere that didn't expand refunds.
```

Comment:
If we're concerned with not having refunds expanded here, should we try to request it if it isn't present? @jclusso tagging you for input

Reply:
I don't think so. I think we should explicitly expand refunds where we need them probably.

Reply:
Does this mean we want to keep this logic in here then?

Reply:
I'm confused.... wouldn't that be the opposite of what I'm saying?

Reply:
You've only said to expand refunds where needed. There either needs to be some handling during a sync on charges or creating local refunds needs to happen elsewhere.

Reply:
Can you review how we use refunds and see if it would be feasible to expand the refunds and sync them specifically in the places we need them.